models/demos/t3000/mixtral8x7b/tt/mixtral_common.py

```diff
@@ -111,102 +116,70 @@ def preprocess_inputs_prefill(
 
-    logger.info(f"# of users: {len(encoded_prompts)}")
-    for i, encoded in enumerate(encoded_prompts):
-        if prefill_seq_len > 0:
-            input_tokens_prefill[i] = torch.tensor(encoded[:prefill_seq_len]).to(input_tokens_prefill)
-        # Right padding
-        input_tokens_decode[i, : len(encoded[prefill_seq_len:])] = torch.tensor(encoded[prefill_seq_len:]).to(
-            input_tokens_decode
-        )
+        input_tokens_prefill.append(input_tokens_prefill_i)
+        # input_tokens_decode.append(input_tokens_decode_i)
 
-    input_mask_bool = input_tokens_decode != tokenizer.pad_id
-    input_mask = input_mask_bool.int()  # from_torch doesn't support bool type
+        decoding_pos.append(len(encoded))
+        prefill_lens.append(prefill_seq_len)
 
     # convert to ttnn tensor
     # Encoded input tokens need to be uint32 for embedding. Otherwise the dtype conversion to bfloat16 will change the tokenizer ID
-    if prefill_seq_len > 0:
-        input_tokens_prefill_tt = [
-            ttnn.from_torch(
-                input_tokens_prefill[i, :].unsqueeze(0),
-                device=device_mesh,
-                dtype=ttnn.uint32,
-                layout=ttnn.ROW_MAJOR_LAYOUT,
-                mesh_mapper=ReplicateTensorToMesh(device_mesh),
-            )
-            for i in range(len(encoded_prompts))
-        ]
-    else:  # Prefill-as-decode for short prompts
-        input_tokens_prefill_tt = None
-
-    input_tokens_decode_tt = [
-        ttnn.from_torch(
-            input_tokens_decode[:, i].unsqueeze(0),
-            device=device_mesh,
-            dtype=ttnn.uint32,
-            layout=ttnn.ROW_MAJOR_LAYOUT,
-            mesh_mapper=ReplicateTensorToMesh(device_mesh),
-        )
-        for i in range(max_prompt_len - prefill_seq_len)
-    ]
-
-    input_mask_tt = [
-        ttnn.from_torch(
-            input_mask[:, i].unsqueeze(0),
-            device=device_mesh,
-            dtype=ttnn.bfloat16,
-            layout=ttnn.ROW_MAJOR_LAYOUT,
-            mesh_mapper=ReplicateTensorToMesh(device_mesh),
-        )
-        for i in range(max_prompt_len - prefill_seq_len)
-    ]
+    # input_tokens_prefill_tt = [
+    #     ttnn.from_torch(
+    #         input_tokens_prefill[i, :].unsqueeze(0),
+    #         device=device_mesh,
+    #         dtype=ttnn.uint32,
+    #         layout=ttnn.ROW_MAJOR_LAYOUT,
+    #         mesh_mapper=ReplicateTensorToMesh(device_mesh),
+    #     )
+    #     for i in range(len(encoded_prompts))
+    # ]
+
+    # input_tokens_decode_tt = [
+    #     ttnn.from_torch(
+    #         input_tokens_decode[:, i].unsqueeze(0),
+    #         device=device_mesh,
+    #         dtype=ttnn.uint32,
+    #         layout=ttnn.ROW_MAJOR_LAYOUT,
+    #         mesh_mapper=ReplicateTensorToMesh(device_mesh),
+    #     )
+    #     for i in range(max_prompt_len - prefill_seq_len)
+    # ]
+
+    input_tokens_prefill_tt = None
```

Comment:
Are these safe to be removed? From here and from everywhere where `preprocess_inputs_prefill()` is called?